macOS向けのWi-Fiのページを最新のmacOS 26 / 15に対応させる (#1990)
utelecon/utelecon.github.io: src/pages/en/utokyo_wifi/macos.mdx

---
title: Using UTokyo Wi-Fi on macOS 26
breadcrumb:
  title: Using UTokyo Wi-Fi on macOS 26
---
import Intro from "./_Intro.astro"
import IssueAccount from "./_IssueAccount.mdx"
import SelectSsid from "./_SelectSsid.astro"
import DeleteProfile from "./_DeleteProfile.astro"
import TroubleConnect from "./_TroubleConnect.mdx"

> <Intro kindOfTerminal="terminal with macOS 26 Tahoe" confirmedTerminal="（MacBook Pro with macOS 26 Tahoe）"/>

Reviewer: 確認したMacはProではなくAirです
```suggestion
<Intro kindOfTerminal="terminal with macOS 26 Tahoe" confirmedTerminal="（MacBook Air with macOS 26 Tahoe）"/>
```

{:#procedure}
### Step 1: Enable the Wi-Fi function from the taskbar
{:#turn-on}
To see the setting screen, select the “System Settings” from the Apple menu on the left side of the menu bar. Select “Wi-Fi” in the side bar and turn on the toggle switch to enable the Wi-Fi function.

> Please note that if the OS version is macOS 12 Monterey or earlier, instead of “System Settings”, “System Preference” will be displayed. The following procedures may vary slightly in different situations, please read the content and procedures for the respective OS version.

Reviewer: 13も非推奨なので12についての言及は不要です。
```suggestion
```

> <figure class="gallery">![](img/macos26-turnon1.png){:.medium.center.border}![](img/macos26-turnon2.png){:.medium.center.border}</figure>

Reviewer: 相対パスにしておきます
```suggestion
<figure class="gallery">![](./img/macos26-turnon1.png){:.medium.center.border}![](./img/macos26-turnon2.png){:.medium.center.border}</figure>
```

{:#procedure}
### Step 1: Enable the Wi-Fi function from the taskbar
{:#turn-on}

> To see the setting screen, select the “System Settings” from the Apple menu on the left side of the menu bar. Select “Wi-Fi” in the side bar and turn on the toggle switch to enable the Wi-Fi function.

Reviewer: ```suggestion
To see the setting screen, select the “System Settings…” from the Apple menu on the left side of the menu bar. Select “Wi-Fi” in the side bar and turn on the toggle switch to enable the Wi-Fi function.
```

### Step 3: Delete the existing Wi-Fi setting information.
{:#delete-profile}
<DeleteProfile  userIdField="Account name"/>

> Select the SSID you found in Step 2, and select the “...” icon on the right side. Then select “Forget This Network” on the displayed screen. When a screen asking 'Forget Wi‑Fi Network “0000UTokyo”?' appears, select “Forget”.

Reviewer: ```suggestion
Select the SSID you found in Step 2, and select the “...” icon on the right side. Then select “Forget This Network…” on the displayed screen. When a screen asking 'Forget Wi‑Fi Network “0000UTokyo”?' appears, select “Forget”.
```

<figure class="gallery">![](./img/macos26-deleteprofile1.png){:.medium.center.border}![](./img/macos26-deleteprofile2.png){:.medium.center.border}</figure>

### Step 4: Enter the required information and connect
{:#create-profile}
The screen of the Wi-Fi settings will be displayed after you select the SSID `0000UTokyo`. To enter the required setting information into each field, follow the instructions below. You do not have to set those settings that are not displayed here. Once you’ve entered the setting information, select “OK” to connect.

| Field name    | Content     |
| :-----------: | :------------------- |
| ID            | Select “None”      |
| Account name       | Enter the User ID for the UTokyo Wi-Fi (begin with a `u` and end with `@wifi.u-tokyo.ac.jp`) |
| Password      | Enter the password for the UTokyo Wi-Fi      |

![](./img/macos26-createprofile.png){:.medium.center.border}

#### Supplement
{:#create-profile-notes}

> * Enter the user ID and password for the UTokyo Wi-Fi account into the fields of “Account name” and “Password”. Note that they are not the 10-digit Common ID and password for the UTokyo Account. You can check the user ID and password in the notification email described in the ”**Preparation**” step or the screen of the completed application.

Reviewer: **Preparation**の部分は日本語版だとリンクになっているので、該当箇所へのリンクを追加したいです。

* If there are any setting items other than mentioned above, please configure them as needed. If you have no particular preferences, you can leave the default settings. For details, please refer to the [Apple Support page](https://support.apple.com/en-us/guide/mac-help/mh11935/mac).

> * During the connecting operation, a screen asking if you trust the certificate may appear. To verify whether the certificate is correct, compare the "server fingerprint" of the displayed certificate with the fingerprint listed in 「[Connect Configuration](/en/utokyo_wifi/#connect-configuration)」. If they match, trust the certificate.

Reviewer: ```suggestion
* During the connecting operation, a screen asking if you trust the certificate may appear. To verify whether the certificate is correct, compare the "server fingerprint" of the displayed certificate with the fingerprint listed in "[Connect Configuration](/en/utokyo_wifi/#connect-configuration)". If they match, trust the certificate.
```

### Step 3: Delete the existing Wi-Fi setting information.
{:#delete-profile}
<DeleteProfile  userIdField="Account name"/>

> Select the SSID you found in Step 2, and select the “...” icon on the right side. Then select “Forget This Network” on the displayed screen. When a screen asking “Forget Wi‑Fi Network “0000UTokyo”?” appears, select “Forget”.

Reviewer: 画像だと…が入っているので追加しておきます
```suggestion
Select the SSID you found in Step 2, and select the “...” icon on the right side. Then select “Forget This Network…” on the displayed screen. When a screen asking “Forget Wi‑Fi Network “0000UTokyo”?” appears, select “Forget”.
```

> Select the SSID you found in Step 2, and select the “...” icon on the right side. Then select “Forget This Network” on the displayed screen. When a screen asking “Forget Wi‑Fi Network “0000UTokyo”?” appears, select “Forget”.

Reviewer: ""のなかに""が入ると少し見ずらいので、外側を''にしました
```suggestion
Select the SSID you found in Step 2, and select the “...” icon on the right side. Then select “Forget This Network” on the displayed screen. When a screen asking 'Forget Wi‑Fi Network “0000UTokyo”?' appears, select “Forget”.
```

> <figure class="gallery">![](img/macos26-deleteprofile1.png){:.medium.center.border}![](img/macos26-deleteprofile2.png){:.medium.center.border}</figure>

Reviewer: ```suggestion
<figure class="gallery">![](./img/macos26-deleteprofile1.png){:.medium.center.border}![](./img/macos26-deleteprofile2.png){:.medium.center.border}</figure>
```

### Step 4: Enter the required information and connect
{:#create-profile}
The screen of the Wi-Fi settings will be displayed after you select the SSID `0000UTokyo`. To enter the required setting information into each field, follow the instructions below. You do not have to set those settings that are not displayed here. Once you’ve entered the setting information, select “OK” to connect.

| Field name    | Content     |
| :-----------: | :------------------- |
| ID            | Select “None”      |
| Account name       | Enter the User ID for the UTokyo Wi-Fi (begin with a `u` and end with `@wifi.u-tokyo.ac.jp`) |
| Password      | Enter the password for the UTokyo Wi-Fi      |

> ![](img/macos26-createprofile.png){:.medium.center.border}

Reviewer: ```suggestion
![](./img/macos26-createprofile.png){:.medium.center.border}
```

#### Supplement
{:#create-profile-notes}

> * Enter the user ID and password for the UTokyo Wi-Fi account into the fields of “Account name” and “Password”. Note that they are not the regular ID and password for the UTokyo Account. You can check the user ID and password in the notification email described in the ”**Preparation**” step or the screen of the completed application.

Reviewer: こういうところに古い共通IDの表記が残ってるんですね
```suggestion
* Enter the user ID and password for the UTokyo Wi-Fi account into the fields of “Account name” and “Password”. Note that they are not the 10-digit Common ID and password for the UTokyo Account. You can check the user ID and password in the notification email described in the ”**Preparation**” step or the screen of the completed application.
```

* If there are any setting items other than mentioned above, please configure them as needed. If you have no particular preferences, you can leave the default settings. For details, please refer to the [Apple Support page](https://support.apple.com/en-us/guide/mac-help/mh11935/mac).
* During the connecting operation, a screen asking if you trust the certificate may appear. To verify whether the certificate is correct, compare the "server fingerprint" of the displayed certificate with the fingerprint listed in 「[Connect Configuration](/en/utokyo_wifi/#connect-configuration)」. If they match, trust the certificate.

>     * In addition, there are two types of fingerprints (SHA-1 and SHA-256) listed on [Connect Configuration](/en/utokyo_wifi/#connect-configuration);matching one type will suffice. ![](img/macos26-certificate.png){:.medium.center.border}

Reviewer: ```suggestion
    * In addition, there are two types of fingerprints (SHA-1 and SHA-256) listed on [Connect Configuration](/en/utokyo_wifi/#connect-configuration);matching one type will suffice. ![](./img/macos26-certificate.png){:.medium.center.border}
```
